qiskit/compiler/assembler.py

```diff
@@ -13,9 +13,15 @@
 
 from qiskit.circuit.quantumcircuit import QuantumCircuit
 from qiskit.compiler.run_config import RunConfig
-from qiskit.qobj import (QasmQobj, QobjExperimentHeader, QobjHeader,
+from qiskit.exceptions import QiskitError
+from qiskit.pulse import ConfiguredSchedule, Snapshot
+from qiskit.pulse.commands import (DriveInstruction, FrameChangeInstruction,
+                                   PersistentValueInstruction, AcquireInstruction)
+from qiskit.qobj import (QasmQobj, PulseQobj, QobjExperimentHeader, QobjHeader,
```

Comment:
This is just a suggestion, but it may be better to just use `from qiskit import qobj` and then call as `qobj.QobjConditional` rather than pollute the namespace with many explicit Qobj class imports. https://google.github.io/styleguide/pyguide.html#22-imports